docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -0,0 +1,196 @@
+//
+//  DOCUMENT IDS:
+//  The reusable content in this file references the following ids which must exist in the document:
+//
+//  - [#create-item-api]
+//  - [#update-item-api]
+//  - [#exception-item-entry-schema]
+//
+// --------------------------------------------------------------------------------------------------
+
+// - - - - - - - - - - - - - - - -
+// A general list of payload properties that must be set when using create/update APIs
+// - - - - - - - - - - - - - - - -
+// tag::endpoint-exceptions-request-payload-requirements[]
+// tag::endpoint-exceptions-request-payload-list-id-and-namespace-requirements[]
+- The `list_id` value must be `pass:a[{endpoint-artifact-list-id}]`.
+- The `namespace_type` value must be `agnostic`.
+// end::endpoint-exceptions-request-payload-list-id-and-namespace-requirements[]
+- The `type` value must be `simple`.
+// end::endpoint-exceptions-request-payload-requirements[]
+
+
+// tag::create-exception-container-request-body[]
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`description` |String |Describes the {endpoint-artifact-name} container. |Yes
+|`list_id` |String a|Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
+|`meta` |Object |Placeholder for metadata about the list container. |No
+|`name` |String |The {endpoint-artifact-name} container's name. |Yes
+|`namespace_type` |String a|Must be set to `agnostic`. |Yes
+|`tags` |String[] |String array containing words and phrases to help categorize the
+{endpoint-artifact-name} container. |No
+|`type` |String a|Must be set to `endpoint`. |Yes
+
+|==============================================
+// end::create-exception-container-request-body[]
+
+
+// tag::create-exception-item-request-body[]
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`comments` |xref:exception-item-comment-schema[comment][] a|Array of xref:exception-item-comment-schema[`comment`] objects. |No, defaults to empty array.
+|`description` |String |Describes the {endpoint-artifact-name-single} item. |Yes
+|`entries` |xref:exception-item-entry-schema[entry][] |Array containing the {endpoint-artifact-name-single} query conditions. See xref:exception-item-entry-schema[] for more details. |Yes.
+|`list_id` |String a|Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
+|`item_id` |String |Unique identifier of the exception item. |No, automatically
+created when it is not provided.
+|`meta` |Object |Placeholder for metadata about the exception item. |No
+|`name` |String |The {endpoint-artifact-name-single} name. |Yes
+|`namespace_type` |String a|Must be set to `agnostic` for endpoint artifacts. |Yes
+ifeval::["{endpoint-artifact-list-id}" == "endpoint_host_isolation_exceptions"]
+|`os_types` |xref:exception-item-os-types[os_types][] |Array containing all xref:exception-item-os-types[os_types]. |Yes
+endif::[]
```

Comment:
required for host isolation exceptions

Reply:
@ashokaditya this seems to be a bug in my PR. `os_type` needs to be set for all artifacts. I will add it to mine. You just need to include an additional note on your host isolation docs to indicate that the item (in create/update) needs to have  all 3 of them. See below